src/DonorDashboards/resources/js/app/components/subscription-manager/payment-method-control/square-control/index.js

```diff
@@ -7,7 +7,7 @@ import './style.scss';
 const cardTokenizeResponse = {};
 let cardBrand = 'unknown';
 
-const SquareControl = ({label, value, forwardedRef, gateway}) => {
+const SquareControl = ({label = null, value, forwardedRef, gateway}) => {
```

Comment:
Do we need a default parameter for value?

Reply:
See https://github.com/impress-org/givewp/pull/7490#issuecomment-2310556555